### PR TITLE
chore: support time range option for evaluation count request

### PR DESCRIPTION
**Pull request by ono-max**

### CODE REVIEW

File: pkg/eventcounter/api/api.go [hunk 1]
```diff
@@ -643,10 +747,31 @@ func getOneDayTimestamps(timestamp time.Time) []int64 {
...
 
-func getDailyTimeStamps(startAt time.Time) []int64 {
-	limit := 31
+func (s *eventCounterService) getTimeStamps(
+	timeRange ecproto.GetEvaluationTimeseriesCountRequest_TimeRange,
+) ([]int64, ecproto.Timeseries_Unit, error) {
+	endAt := time.Now()
+	switch timeRange {
+	case ecproto.GetEvaluationTimeseriesCountRequest_TWENTY_FOUR_HOURS:
+		startAt := getTwentyFourHoursAgo(s.location, endAt)
```
[the remark this answers is not in the document]
Comment (ono-max) [reply pkg/eventcounter/api/api.go:756]: I refactor the logic since defining getTwentyFourHoursAgo is not cool. Thank you!

**Comment by ono-max** [comment]
@cre8ivejp 

Updated. Thank you.

File: pkg/eventcounter/api/api.go [hunk 2]
```diff
@@ -62,7 +63,8 @@ const (
 )
 
 var (
-	jpLocation = time.FixedZone("Asia/Tokyo", 9*60*60)
+	jpLocation          = time.FixedZone("Asia/Tokyo", 9*60*60)
+	errUnknownTimeRange = errors.New("")
```
Comment (cre8ivejp) [inline pkg/eventcounter/api/api.go:67]: The error message is empty.

**Review by cre8ivejp — APPROVED** [review]
Thank you!

**Pull request merged**